capstone/src/main/java/com/google/sps/servlets/GettingDetails.java

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.maps.GaeRequestHandler;
+import com.google.maps.GeoApiContext;
+import com.google.maps.FindPlaceFromTextRequest;
+import com.google.maps.PlacesApi;
+import com.google.maps.model.PlaceDetails;
+import com.google.maps.model.PlacesSearchResult;
+
+public final class GettingDetails {  
```

Comment:
another small thing, but because this class isn't a Servlet, you could move it to capstone/src/main/java/com/google/sps/data/

Reply:
I'd also recommend creating a standard approach to service names. Something along the lines of RestaurantDetailFetcher if you are grabbing the information from an external api, or RestaurantDetailGetter if it's internal.